docker-compose-localdev.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.1'
+
+services:
+
+  wordpress:
+    volumes:
+      - ./wordpress:/var/www/html
```

Comment:
Does the `-f` work the same as `docker-compose.override.yml`? If it does, we should only need to configure the `/var/www/html` mount point, rather than duplicating all of them.

Reply:
You're absolutely right, have corrected this :)